app/assets/images/payment_icons/kbc_cbc.svg

```diff
@@ -0,0 +1 @@
+<svg width="38" height="24" viewBox="0 0 38 24" role="img" fill="none" xmlns="http://www.w3.org/2000/svg" aria-labelledby="pi-kbc_cbc"><title id="pi-kbc_cbc">KBC/CBC</title><rect width="38" height="24" rx="2" fill="#fff"/><rect x=".5" y=".5" width="37" height="23" rx="1.5" stroke="#000" stroke-opacity=".07"/><path fill-rule="evenodd" clip-rule="evenodd" d="M18.663 11a3 3 0 100-6 3 3 0 000 6z" fill="#019ACC"/><path fill-rule="evenodd" clip-rule="evenodd" d="M17.952 11.742a3.898 3.898 0 01-2.364-.805c-3.205.336-5.588.812-5.588.812v1.247h17.032V10.37s-2.945-.018-6.27.16a3.858 3.858 0 01-2.81 1.212z" fill="#019ACC"/><path fill-rule="evenodd" clip-rule="evenodd" d="M12.02 19H10v-5h2.02v2.25l1.2-2.25h2.421l-1.773 2.584L15.541 19h-2.31l-1.198-1.88L12.02 19zM27 18.735a5.093 5.093 0 01-1.622.265C23.43 19 22 18.349 22 16.455 22 14.744 23.386 14 25.281 14c.527 0 1.196.068 1.719.265v1.658c-.389-.277-.776-.449-1.332-.449-.722 0-1.445.419-1.445 1.026 0 .61.72 1.027 1.445 1.027.547 0 .944-.178 1.331-.442l.001 1.65zM18.02 16.119h1.005c.389 0 .541-.187.541-.474 0-.33-.153-.494-.513-.494H18.02v.968zm0 1.864h1.052c.395 0 .53-.194.53-.48 0-.288-.136-.494-.525-.494h-1.058v.974zM16 14h4.082c1.06 0 1.408.49 1.408 1.253 0 .855-.642 1.226-1.302 1.268v.014c.678.063 1.338.154 1.338 1.162 0 .659-.348 1.303-1.514 1.303H16v-5z" fill="#003466"/></svg>
```

Comment:
The border radius doesn't look the same as other icons: https://screenshot.click/02-13-ucbtl-mexf4.mp4